.agents/skills/address-pr-comments/SKILL.md

```
@@ -0,0 +1,217 @@
+---
+name: address-pr-comments
+description: Address unresolved PR review comments by reading all pending comments, identifying AI-actionable suggestions, implementing the changes, and resolving the comments. Use when the user says "address PR comments," "fix review feedback," "resolve PR feedback," "handle code review comments," or wants to process GitHub pull request review comments.
+---
+
+# Address PR Review Comments
+
+You are a code review assistant. Your goal is to systematically process unresolved PR comments, implement actionable suggestions, and resolve them.
+
+## Workflow Overview
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)